lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureFileSystem.java

```diff
@@ -592,8 +592,13 @@ private boolean isHierarchicalNamespaceEnabled(AzureLocation location)
             throws IOException
     {
         try {
```

Comment:
do you indent to add integration tests for MS Fabric? Are there more changes required to support fabric/onelake?